gcn/notices/fermi/gbm/strict/alert.1.example.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/fermi/gbm/alert.schema.json",
+  "additional_info": "*** ABOUT THIS EXAMPLE: This is an example of a Fermi GBM Alert GCN notice. All fields, including the optional ones, are present. This 'about' note is not going to be part of a streamed real notice. This field may be missing or empty in case of no comments. An example of a typical notice comment follows. ***\n\nLightcurve and other quicklook files will not be created until about 15 minutes after the trigger.",
+  "alert_datetime": "2024-08-24T02:14:54",
+  "alert_tense": "current",
+  "alert_type": "initial",
+  "data_archive_page": "http://heasarc.gsfc.nasa.gov/FTP/fermi/data/gbm/triggers/2024/bn240812094",
+  "detectors_status": [
+    "triggered",
+    "on",
+    "triggered",
+    "triggered",
+    "on",
+    "on",
+    "triggered",
+    "on",
+    "on",
+    "on",
+    "on",
+    "on",
+    "on",
+    "on"
+  ],
+  "event_name": ["GRB240824A"],
+  "hitl": false,
```

Comment:
Please make the example field name consistent with schema.
Additionally, validate the schema against examples, in case any change is left out.

Reply:
@Vidushi-GitHub: Changed. Thank you for the reminder!